Quintiles config (#389)
medic/cht-docs: content/en/apps/guides/forms/wealth-quintiles.md

---
title: "Tracking Wealth Quintiles"
linkTitle: "Quintiles"
weight: 
description: >
  How to track wealth quintiles on the profile of each family member in the household
relatedContent: >

---

> 
> A few things are needed to get the data copied onto the family members. 
> 
> 1. A report submitted with either `doc.fields.NationalQuintile` or `doc.fields.UrbanQuintile` or both. 
> 1. An offline user that has the permission of `can_write_wealth_quintiles`. 
> 1. Once the form is submitted the contact will then get the fields `"wealth_quintile_national"` and ` "wealth_quintile_urban"` if they are provided in the form.
> 

Reviewer: Additional context would be helpful here. Also, I think this can be simplified a bit:

```suggestion
Household surveys with questions about the home, possessions, and access to safe drinking water have been used to create equity score and improve the targeting of health services. This guide will cover how quintile information from a household survey can be used in customizing care for individual household members. For example, the equity quintile can be used to increase the number of pregnancy follow-ups for women in specific households, or to display specific notes or questions within patient forms.

For quintile information from a household survey to be used for people in that household, a form about a place must contain `NationalQuintile` and/or `UrbanQuintile` fields at the top level. When that form is submitted all the people belonging to that place will get the corresponding `wealth_quintile_national` and `wealth_quintile_urban` fields. Those fields can then be used by forms, tasks, and contact profiles accordingly.

Note that the `can_write_wealth_quintiles` permission is needed for the user that is submitting the form.
```

### Sample Report

> 

Reviewer: I think a sample XLSForm would be nice here.

Author: added

``` json
{
  "form": "family_survey",
  "type": "data_record",
  "content_type": "xml",
  "fields": {
    "place_id": "23c7ab6e-6ea5-4f9f-9ffa-f00a1e30278d",

>     "place_name": "Medic Area",

Reviewer: The `place_name` isn't required and it may be confusing here.
```suggestion
```

Author: 👍

    "NationalQuintile": "5",

>     "UrbanQuintile": "3"

Reviewer 2: Ideally would these be strings, numbers, or ints?

Author: I'm not 100% sure as I just used a form that was already built. I'm going to generate a sample form as requested above and try the different data types.

Author: @garethbowen  setting the type on the xlsform doesn't set the actual type. It always seems to be a string. I had prompted a question about storing the specific type to the report but did not get an answer. Do we have a way to store the data in a specific type?

Reviewer 2: Not sure... Don't let me block this PR. If I find a way I can update later...

---
title: "Tracking Wealth Quintiles"
linkTitle: "Quintiles"
weight: 
description: >
  How to track wealth quintiles on the profile of each family member in the household
relatedContent: >

---

> Introduced in 2.16.0

Reviewer: Other pages have this text italicized, so doing the same here for consistency.

```suggestion
_Introduced in v2.16.0_
```

Household surveys with questions about the home, possessions, and access to safe drinking water have been used to create equity score and improve the targeting of health services. This guide will cover how quintile information from a household survey can be used in customizing care for individual household members. For example, the equity quintile can be used to increase the number of pregnancy follow-ups for women in specific households, or to display specific notes or questions within patient forms.

For quintile information from a household survey to be used for people in that household, a form about a place must contain `NationalQuintile` and/or `UrbanQuintile` fields at the top level. When that form is submitted all the people belonging to that place will get the corresponding `wealth_quintile_national` and `wealth_quintile_urban` fields. Those fields can then be used by forms, tasks, and contact profiles accordingly.

Note that the `can_write_wealth_quintiles` permission is needed for the user that is submitting the form.

### Sample form

| type | name | label | relevant | appearance | calculate | ... |
|---|---|---|---|---|---|---|				

> |begin group |	inputs |	Patients |||		./source = 'user'	field-list
> |string |	source |	Source | | hidden
> |string |	source_id |	Source ID | hidden |	
> |begin group |	contact |	NO_LABEL|				
> |db:person |	_id |  |	 |		db-object 
> |begin | group |	parent | NO_LABEL 			
> |begin group |	contact	| NO_LABEL		
> |string |	phone |	CHW Phone | | hidden
> |string |	name |	CHW Name | | hidden	
> |end group |						
> |end group |					
> |end group |						
> |end group |						
> |integer|	NationalQuintile|	National Quintile|		
> |integer|	UrbanQuintile	| Urban Quintile				
> |calculate|	place_id|	ID|	||			../inputs/contact/_id|
> 

Reviewer: I think it would be clearer to replace the boilerplate with a select field.
```suggestion
| string | place_id | | | select-contact type-household
| integer | NationalQuintile | National Quintile		
| integer | UrbanQuintile | Urban Quintile
```
